scripts/build_mod_chip.py

```diff
@@ -251,6 +251,14 @@
         "pin_shape": (-0.5, 1.6),
     },
 
+    # Coilcraft XFL4020 series Inductors
+    "XFL4020": {
+        "pad_shape": (0.98, 3.8),
```

Comment:
I read 3.4mm for the pad length in the datasheet? http://www.coilcraft.com/pdfs/xfl4020.pdf